templates/adp-backend-template-node/skeleton/.secrets.baseline

```diff
@@ -104,6 +107,16 @@
       "path": "detect_secrets.filters.heuristic.is_templated_secret"
     }
   ],
-  "results": {},
-  "generated_at": "2022-02-22T08:52:45Z"
+  "results": {
+    "docker-compose.yaml": [
+      {
+        "type": "Secret Keyword",
+        "filename": "docker-compose.yaml",
+        "hashed_secret": "b3054ff0797ff0b2bbce03ec897fe63e0b6490e0",
```

Comment:
should this be checked in?

Reply:
@kbabigumira it is just on the docker-compose file for local development. There isn't anything sensitive on it. As per recommendation, we can set "is_verified": true to the detected secret.